packages/connection/src/class-manager.php

```diff
@@ -2780,6 +2775,22 @@ public function get_signed_token( $token ) {
 	 */
 	public function add_stats_to_heartbeat( $stats ) {
 
+		// Connected but missing the connection owner?
+		if ( $this->is_connected() && $this->is_missing_connection_owner() ) {
```

Comment:
Before, we were still sending a `0` value if there wasn't a missing owner at all. Doing this also serves some purpose, as it tells us what % of total heartbeats are missing it, which is hard to get by just sending the failing cases. 

I'm also thinking whether or not we should create a new bin for these checks, as adding to the ones existing will conflate old jetpack versions sending not the same data. 

Thoughts?

Reply:
Perhaps it makes sense to use a separate bin for all heartbeat stats that go through the _package_. `jetpack-v2` as a prefix could continue to be for jetpack-specific stats though.

Reply:
> Before, we were still sending a 0 value if there wasn't a missing owner at all. 

I removed that intentionally and forgot to mention it. I get your point... I felt like it was overlapping with other stats, because this was basically a ping identifying a healthy connection... 

> Perhaps it makes sense to use a separate bin for all heartbeat stats that go through the package. jetpack-v2 as a prefix could continue to be for jetpack-specific stats though.

The `jetpack` and the `v2` prefixes are automatically added by the `a8c-mc-stats` and `heartbeat` packages respectively. We'd have to modify them to allow that. I have to look into it.

> Thoughts?

Maybe we should think of a more generic "connection health" bucket and put everything there